Task_2/src/model/Component.java

```diff
@@ -0,0 +1,13 @@
+package model;
+
+import java.util.List;
+
+public interface Component<T> {

```

Comment:
Is it really need to be parameterized?

Reply:
I agree, this is not necessary here. I thought I’ll have to use a generic here